crates/sui-types/src/gas_model/gas_v2.rs

```diff
@@ -144,6 +144,20 @@ mod checked {
         }
     }
 
+    #[derive(Debug)]
+    pub struct PerObjectStorage {
+        /// storage_cost is the total storage gas to charge. This is computed
+        /// at the end of execution while determining storage charges.
+        /// It tracks `storage_bytes * obj_data_cost_refundable` as
+        /// described in `storage_gas_price`
+        /// It will be multiplied by the storage gas price.
```

Comment:
Is this value before or after multiplying the gas price? The comment is a bit confusing

Reply:
this value is before multiplying

Reply:
updated